internal/controller/common/helper.go

```diff
@@ -573,6 +573,36 @@ func GetDBMonitoringConfig(
 	return monitoring, nil
 }
 
+// GetPodSchedulingPolicy returns the PodSchedulingPolicy object by name.
+func GetPodSchedulingPolicy(ctx context.Context, c client.Client, pspName string) (*everestv1alpha1.PodSchedulingPolicy, error) {
+	psp := &everestv1alpha1.PodSchedulingPolicy{}
+	if pspName != "" {
+		if err := c.Get(ctx, types.NamespacedName{
+			Namespace: SystemNamespace,
```

Comment:
To avoid re-introducing this hard-coded namespace and potentially making it harder to fix https://github.com/percona/everest/issues/1159 do you think it would be okay to use the namespace where the operator is running instead?